### ISSUE TITLE
Cannot initialize CUDA

### ISSUE THREAD

**Issue by Space-B** [OpenDroneMap/ODM#1472]
### How did you install OpenDroneMap?

Native windows installer

### What's your browser and operating system?

Waterfox 91 on Windows 11

### What is the problem?

GPU acceleration falls back to the CPU
```
[INFO] Photo dimensions for feature extraction: 1024px
[INFO] CUDA drivers detected
[WARNING] Cannot use GPU for feature extraction: Cannot initialize CUDA
```
### What should be the expected behavior?

CUDA is used for feature extraction

### How can we reproduce this?

Attempt to process any dataset with an NVIDIA GPU that supports CUDA. 
Others having the same issue with multiple different NVIDIA GPUs in [this](https://community.opendronemap.org/t/native-odm-windows-cannot-initialize-cuda/11899) forum post. This issue happens for me with CUDA 11.7 and ODM 2.8.5.

**Comment by martinr36** [comment]
Seeing the same here:

How did you install OpenDroneMap?

Native windows installer
What's your browser and operating system?

Waterfox G4.1.2.1 on Windows 11

**Comment by pierotofy** [comment]
Thanks for the reports! :pray: It should be fixed with #1475
